website/docs/guides/orchestration/webhooks/zapier-refresh-tableau-workbook.md

```diff
@@ -0,0 +1,160 @@
+See [Create a webhook subscription](/docs/deploy/webhooks#create-a-webhook-subscription) for full instructions. Your event should be **Run completed**, and you need to change the **Jobs** list to only contain any jobs whose completion should trigger a report refresh.
+
+Make note of the Webhook Secret Key for later.  Paste in the webhook URL obtained from Zapier in step 2 into the "Endpoint" field and test the endpoint.
+
+Once you've tested the endpoint in dbt Cloud, go back to Zapier and click **Test Trigger**, which will create a sample webhook body based on the test event dbt Cloud sent.
+
+The sample body's values are hard-coded and not reflective of your project, but they give Zapier a correctly-shaped object during development. 
+
+### 4. Store secrets 
+In the next step, you will need the Webhook Secret Key from the prior step, and your Tableau authentication credentials and details.  Specifically, you'll need your Tableau server/site URL, server/site name, PAT name, and PAT secret.
+
+Zapier allows you to [store secrets](https://help.zapier.com/hc/en-us/articles/8496293271053-Save-and-retrieve-data-from-Zaps), which prevents your keys from being displayed in plaintext in the Zap code. You will be able to access them via the [StoreClient utility](https://help.zapier.com/hc/en-us/articles/8496293969549-Store-data-from-code-steps-with-StoreClient).
+
+This guide assumes the names for the secret keys are: `DBT_WEBHOOK_KEY`, `TABLEAU_SITE_URL`, `TABLEAU_SITE_NAME`, `TABLEAU_API_TOKEN_NAME`, and `TABLEAU_API_TOKEN_SECRET`. If you are using different names, make sure you update all references to them in the sample code.
+
+This guide uses a short-lived code action to store the secrets, but you can also use a tool like Postman to interact with the [REST API](https://store.zapier.com/) or create a separate Zap and call the [Set Value Action](https://help.zapier.com/hc/en-us/articles/8496293271053-Save-and-retrieve-data-from-Zaps#3-set-a-value-in-your-store-0-3).
+
+#### a. Create a Storage by Zapier connection
+If you haven't already got one, go to <https://zapier.com/app/connections/storage> and create a new connection. Remember the UUID secret you generate for later. 
+
+#### b. Add a temporary code step
+Choose **Run Python** as the Event. Run the following code: 
+```python 
+store = StoreClient('abc123') #replace with your UUID secret
+store.set('DBT_WEBHOOK_KEY', 'abc123') #replace with your dbt Cloud Webhook key
+store.set('TABLEAU_SITE_URL', 'abc123') #replace with your Tableau Site URL, inclusive of https:// and .com
+store.set('TABLEAU_SITE_NAME', 'abc123') #replace with your Tableau Site/Server Name
+store.set('TABLEAU_API_TOKEN_NAME', 'abc123') #replace with your Tableau API Token Name
+store.set('TABLEAU_API_TOKEN_SECRET', 'abc123') #replace with your Tableau API Secret
+```
+Test the step. You can delete this Action when the test succeeds. The key will remain stored as long as it is accessed at least once every three months.
```

Comment:
```suggestion
#### b. Add a temporary code step

Choose **Run Python** as the Event and run the following code: 

```python 
store = StoreClient('abc123') #replace with your UUID secret
store.set('DBT_WEBHOOK_KEY', 'abc123') #replace with your dbt Cloud Webhook key
store.set('TABLEAU_SITE_URL', 'abc123') #replace with your Tableau Site URL, inclusive of https:// and .com
store.set('TABLEAU_SITE_NAME', 'abc123') #replace with your Tableau Site/Server Name
store.set('TABLEAU_API_TOKEN_NAME', 'abc123') #replace with your Tableau API Token Name
store.set('TABLEAU_API_TOKEN_SECRET', 'abc123') #replace with your Tableau API Secret
```
Test the step. You can delete this Action when the test succeeds. The key will remain stored as long as it is accessed at least once every three months.
```